functions.inc.php

```diff
@@ -162,24 +161,26 @@ function csrf_validate($given = "") {
 	return $_SESSION['csrf'] == $given;
 }
 
-//Random string generation - code from http://stackoverflow.com/questions/4356289/php-random-string-generator (CC license)
+// Random string generation - code from
+// http://stackoverflow.com/questions/4356289/php-random-string-generator (CC license)
 function generateRandomString($length = 10) {
-    $characters = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
-    $charactersLength = strlen($characters);
-    $randomString = '';
-    for ($i = 0; $i < $length; $i++) {
-        $randomString .= $characters[rand(0, $charactersLength - 1)];
-    }
-    return $randomString;
+	$characters = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+	$charactersLength = strlen($characters);
+	$randomString = '';
+	for ($i = 0; $i < $length; $i++) {
+		$randomString .= $characters[rand(0, $charactersLength - 1)];
+	}
+	return $randomString;
 }
 
 //getting user's IP
 function get_client_ip_env() {
 	$ipaddress = '';
-	if(getenv('REMOTE_ADDR'))
+	if(getenv('REMOTE_ADDR')) {
 		$ipaddress = getenv('REMOTE_ADDR');
-	else
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method get_client_ip_env uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://www.codacy.com/app/naczelnik/ambasadorzy-siecobywatelska-pl/file/1624604487/issues/source?bid=2991913&fileBranchId=3033149#l181)